packages/core/src/physics/PhysicsMaterial.ts

```diff
@@ -6,9 +6,9 @@ import { PhysicsMaterialCombineMode } from "./enums/PhysicsMaterialCombineMode";
  * Material class to represent a set of surface properties.
  */
 export class PhysicsMaterial {
-  private _bounciness: number = 0.1;
-  private _dynamicFriction: number = 0.1;
-  private _staticFriction: number = 0.1;
+  private _bounciness: number = 0;
+  private _dynamicFriction: number = 0;
+  private _staticFriction: number = 0;
```

Comment:
Do you think 0 is the best default value?

Reply:
changed to 0.5